Makefile

```diff
@@ -32,6 +32,7 @@ unit-tests:
 		pip3 install --no-cache-dir -r code-env/python/spec/requirements.txt; \
 		export PYTHONPATH="$(PYTHONPATH):$(PWD)/python-lib"; \
 		export DICTIONARY_FOLDER_PATH="$(PWD)/resource/dictionaries"; \
+		export STOPWORDS_FOLDER_PATH="$(PWD)/resource/stopwords"; \
```

Comment:
You need to make tests to test. Not sure what the value is to write custom make rules to test individually?

Reply:
I was not thinking about custom make rules. Actually, my idea was to be able to test in python directly the test function, it can be useful to run the test that we know is working and to be able to debug it for example outside of DSS. Also let's imagine somebody wants to use the python-lib outside of DSS, then he will probably not think of settings the env variable but just to reuse existing functions out of the box.

Reply:
For now, I don't really envision using the items in python-lib outside of DSS. Pytest already takes care of running all the test_... files under the tests/unit/folder. While I can easily write custom make rules for each test files, it's a lot of additional make rules. Not sure what the value is?

Reply:
Maybe it is more a personal trick that I use, the ability to test the python lib outside of DSS is very useful because as a reviewer you can run and debug them running a test and have a look at intermediary objects. Again, I don't recommend creating additional make rules.

Reply:
I think you should talk with Mayeul about the global env variables in tests, maybe he will have a more knowledgeable opinion

Reply:
Actually, this usage of variables in the the makefile was introduced by Mayeul when we worked on the test infra a few months ago.

Reply:
Our assumption is to really have one test rule in the make file which can cover all tests cases across plugins, and send results to Jenkins in a consistent way.